<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVS Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnNewClearance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New Clearance Verification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewMinimum">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New Minimum Install Distance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSystemCalibration">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>System Calibration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadExisting">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Existing Profile</string>
    </property>
   </widget>
   <widget class="QListView" name="lstExistingProfiles">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>411</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
